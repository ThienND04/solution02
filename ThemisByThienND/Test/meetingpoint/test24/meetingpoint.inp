184 292 13
42 31 690905
119 181 137572
40 15 326332
88 176 874433
17 55 735649
182 119 26227
56 91 326486
62 160 114659
102 36 672501
52 77 362220
139 102 19374
24 18 50727
42 2 1000000
181 4 68232
159 42 396590
144 11 77653
82 155 132031
44 136 331984
4 19 883052
140 21 799112
16 69 695694
108 164 48296
66 30 859548
80 134 161166
100 54 161627
4 102 136835
107 135 831065
162 5 7603
169 174 59224
35 23 1874
173 14 531
167 173 142559
159 127 525518
123 5 9123
88 150 67122
134 23 207899
117 122 753113
178 164 26886
116 114 439377
160 65 291959
72 16 415311
149 1 151446
174 167 783061
103 158 92428
145 15 522796
21 124 11346
151 37 69998
90 25 375737
14 168 630772
59 132 34818
45 12 95022
102 75 71319
170 63 216559
56 46 204059
184 10 288223
35 135 12433
76 97 775175
121 170 696586
54 62 85232
165 46 30392
165 118 551561
8 162 13061
104 143 931834
4 38 890833
170 116 624899
168 7 293290
155 20 153363
122 9 48611
62 141 21059
96 26 316334
52 3 975710
129 84 415940
42 184 6558
31 139 670488
12 55 914160
87 86 679173
21 148 249834
180 43 657430
102 184 7600
48 99 148194
147 70 122869
45 4 61050
142 164 287441
42 180 204019
49 158 37127
184 164 123130
180 128 50161
86 139 213636
58 125 701842
58 54 122605
107 10 590127
158 151 23859
42 49 224778
103 115 76958
48 94 428604
33 64 411758
56 137 239078
63 28 477578
101 20 811767
62 88 38277
73 16 380263
144 64 103851
172 4 788844
176 77 517380
165 64 678304
82 171 493089
105 164 124399
20 84 550017
45 174 751872
130 25 12526
156 163 324378
133 66 67969
135 50 342722
115 130 10490
157 130 407405
161 149 820127
184 106 235686
180 168 452464
156 115 664591
173 143 29488
119 6 38727
82 60 118360
169 94 609981
168 147 335495
10 113 106424
111 119 458256
24 14 5136
71 36 262251
16 131 154110
166 83 629457
106 62 214979
62 35 14145
143 65 366099
127 47 744917
146 105 94426
100 60 11591
176 107 55066
95 168 239879
68 59 21298
58 56 473255
22 34 23422
68 54 34810
53 86 788667
22 99 26324
5 37 216348
7 89 240887
67 115 390920
173 102 24137
59 74 26812
145 8 101889
114 127 217717
99 108 693529
135 154 981064
157 104 571322
130 9 54404
92 102 660250
22 72 463508
184 25 1806
52 135 6530
83 180 169663
139 121 172910
8 78 183296
54 149 781563
153 19 906808
58 146 75441
25 104 991255
141 90 335777
69 18 620020
42 26 685172
62 109 165022
34 83 534458
44 29 72085
79 7 63377
8 130 36722
109 115 160906
3 78 795175
143 93 69253
62 153 99974
12 115 85641
157 96 46342
166 45 804211
42 29 12395
21 119 720940
6 20 62926
179 158 29313
20 88 53139
47 130 169415
172 67 517537
154 165 92099
169 135 850850
83 98 103953
77 177 278596
152 24 48101
39 164 806471
85 88 332855
88 41 892040
164 107 755221
117 141 828696
53 52 975462
151 43 649939
73 161 325348
131 29 147617
135 146 105818
1 82 641431
144 164 30963
24 96 337989
93 7 825853
24 180 173171
161 111 604657
46 178 755840
16 161 54914
9 122 48611
122 183 39905
164 13 129233
174 82 559841
29 24 18452
36 162 615984
48 71 298911
181 141 34438
118 21 550029
16 57 270607
106 17 53290
79 95 116788
173 109 153991
9 120 812517
106 62 214979
30 75 806518
131 106 69115
135 27 655733
56 137 239076
20 19 915366
40 124 56010
28 74 13473
106 81 674961
184 170 896471
12 109 75264
181 56 625595
113 101 529242
109 71 232121
48 33 47057
14 51 345983
157 144 267645
52 33 615887
110 17 736743
27 53 326259
107 112 137945
64 126 159743
89 115 690622
153 118 264007
76 161 112612
173 126 66462
151 6 43019
66 126 72313
136 57 139955
94 103 186375
84 152 576631
160 99 729589
93 108 40946
46 117 35394
29 62 1752
93 76 49509
78 77 181684
128 121 60854
98 58 47743
98 172 656992
78 180 23774
50 180 179429
16 102 4860
116 162 207454
110 92 357869
141 140 93837
155 56 436313
61 88 631862
12 175 600861
38 93 904789
61 164 567717
76 148 504895
149 32 34037
178 159 253130
173 145 121207
88 130 44650
176 63 253505
83 21 554475
176 179 709893
162 129 182082
99 181 788751
62 66 5183
30 31 187975
128 142 149831
150 138 570154
163 110 11929
70 45 670182
283915 68745
48167 390907
587438 550810
489345 374155
385557 471565
451672 379930
999438 822341
540819 732604
587512 366879
952453 533922
925823 655863
379878 840936
714770 58553
