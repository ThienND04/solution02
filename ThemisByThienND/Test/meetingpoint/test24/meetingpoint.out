61011187500
44642379775
376800943778
313881052695
247308212067
289716422632
641070515778
346898071989
325605112500
473855775000
582078412500
288627885498
60833405138
